Cargo.toml

```diff
@@ -14,18 +14,24 @@ rust-version = "1.67.0"
 
 [[example]]
 name = "simple"
-required-features = ["inspect"]
 
 [[example]]
 name = "multiple"
-required-features = ["inspect"]
 
 [features]
-default = ["serialize"]
-inspect = ["bevy-inspector-egui"]
-serialize = ["serde"]
+default = ["serde"]
+serde = ["dep:serde"]
 
 [dependencies]
-bevy = { version = "0.11" }
-bevy-inspector-egui = { version = "0.19", optional = true }
+bevy = { version = "0.11", default-features = false, features = [
+    "bevy_render",
+    "bevy_ui"
+] }
+serde = { version = "^1", optional = true }
+
+[build-dependencies]
```

Comment:
Although I think the idea is great, I don't think it should go as a build dependency, because sometimes someone might need it in production or in some other way, so I think it should be kept as a feature base

Reply:
If they need the derive feature, they could just add it themselves? Why should we force it on them?